docs/fastdds/xml_configuration/common.rst

```diff
@@ -351,6 +352,55 @@ Please refer to :ref:`threadsettingsqos` for further information on ResourceLimi
      - ``int32_t``
      - -1
 
+.. _BuiltinTransportType:
+
+BuiltinTransports
+^^^^^^^^^^^^^^^^^
+
+It controls the builtin transports that will be used during the initialization of the DomainParticipant.
+Please refer to :ref:`rtps_layer_builtin_transports` for further information on builtin transports.
+
+This tag can be configured with the following attributes:
+
+.. list-table::
+  :header-rows: 1
+  :align: left
+
+  * - Name
+    - Description
+    - Values
+    - Default
+  * - ``max_msg_size``
+    - Maximum message size that will be specified in |br|
+      the transport layer.
+      Valid values: from 1 to 2^32-1.
+    - ``uint32_t``
+    - 65500
+  * - ``sockets_size``
+    - Size of the send and receive socket buffers.
+      Valid values: from 0 to 2^64-1.
+    - ``uint64_t``
+    - OS default
+  * - ``non_blocking``
+    - Whether to use non-blocking send calls or not.
+      Valid values: true or false.
+    - ``bool``
+    - false
+
+This type must contain the following tag:
+
+.. list-table::
+   :header-rows: 1
+
+   * - Name
+     - Description
+     - Values
+     - Default
+   * - ``<mode>``
+     - Defines the builtin transport mode.
+     - ``string_255``
+     - ``DEFAULT``
+
```

Comment:
I would include this section right after the previous one.
the other would start:
_"The ``<builtinTransport>`` tag can be ..."_


```suggestion
This type must follow this configuration:

.. list-table::
   :header-rows: 1

   * - Name
     - Description
     - Values
     - Default
   * - ``<builtinTransport>``
     - Defines the builtin transport mode.
     - ``NONE`` <br \> ``DEFAULT`` <br \> ``DEFAULTv6`` <br \> ``SHM`` <br \> ``UDPv4`` <br \> ``UDPv6`` <br \>  ``LARGE_DATA`` <br \>  ``LARGE_DATAv6``
     - ``DEFAULT``

```